packages/backend/src/payment-method/ilp/connector/core/middleware/account.ts

```diff
@@ -87,10 +87,6 @@ export function createAccountMiddleware(serverAddress: string): ILPMiddleware {
               LiquidityAccountType.INCOMING
             )
           }
-          ctx.services.logger.debug(
```

Comment:
logs were noisy, especially during quoting step (since we would get 10 of these)